treeherder/perf/auto_perf_sheriffing/backfill_reports.py

```diff
@@ -155,6 +156,17 @@ def _has_relevant_platform(self, alert: PerformanceAlert):
     def _extract_by_relevant_platforms(self, alerts):
         return list(filter(self._has_relevant_platform, alerts))
 
+    def _is_untriaged(self, alert: PerformanceAlert):
+        """
+        Filter criteria based on the 'untriaged' status.
+        """
+        if alert.status == PerformanceAlert.UNTRIAGED:
+            return True
+        return False
+
+    def _extract_untriaged_alerts(self, alerts):
+        return list(filter(self._is_untriaged, alerts))
```

Comment:
We can move the filtering upper and filter directly in the db:

https://github.com/mozilla/treeherder/blob/98faf9cc5dbff103d7e3c3784c04d701d83a07ae/treeherder/perf/auto_perf_sheriffing/backfill_reports.py#L324